proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -341,6 +341,18 @@ enum BadDataBehavior {
   POISON = 3;
 }
 
+// Directives for how to handle {@code null} and {@code NaN} values
+enum UpdateByNullBehavior {
+  // In the case of Current - null, the null dominates so Column[i] - null = null
+  NULL_DOMINATES = 0;
```

Comment:
I think we prefer the 0 entry for an enum to be a "this isnt specified" value, rather than require that defaults can never change (which would break the wire format).